src/rules/trailingCommaRule.ts

```diff
@@ -207,7 +207,7 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
                         (node as ts.CallExpression | ts.NewExpression).arguments!,
                         node.end,
                         "functions",
-                        noRest
+                        isSpreadParameter
```

Comment:
what's wrong with simply passing `isSpreadElement` here? the function below seems like a passthrough.